### PR TITLE
Ux tweaks

### PR DESCRIPTION
**Pull request by StevePole**
image

image

### CODE REVIEW

File: src/identity/IdentityPage.test.tsx [hunk 1]
```diff
@@ -443,6 +451,14 @@ describe('Identity page', () => {
   });
 });
 
+function getTestResultsLink() {
+  return screen.getByTestId('test-result-link');
```
Comment (oliverviljamaa) [inline src/identity/IdentityPage.test.tsx:455]: Any reason to not use the text?

File: src/identity/IdentityPage.tsx [hunk 2]
```diff
@@ -91,19 +86,29 @@ export const IdentityPage = () => {
...
             </NavLink>
-            <NavLink as={RouterNavLink} to={`${url}/tests`} variant="tab">
-              <TestIcon mr={1} /> Tests
+            <NavLink
+              as={RouterNavLink}
+              to={`${url}/profile`}
+              variant="tab"
+              data-testid="share-access-link"
+            >
+              <ProfileIcon mr={1} /> Share
```
Comment (oliverviljamaa) [inline src/identity/IdentityPage.tsx:111]: 👏

File: src/identity/IdentityPage.test.tsx (hunk 1 above)
Comment (StevePole) [reply src/identity/IdentityPage.test.tsx:455]: 'results' appears in two places on the page, so it finds two different elements.

Comment (oliverviljamaa) [reply src/identity/IdentityPage.test.tsx:455]: We should be able to `getByText('Results')`, though  — for a title, exact match querying is more acceptable.

**Pull request merged**